e2e/cypress/integration/files_and_attachments/upload_files_spec.js

```diff
@@ -179,6 +179,33 @@ describe('Upload Files', () => {
             cy.wait(TIMEOUTS.HALF_SEC);
         });
     });
+
+    it('MM-T337 CTRL/CMD+U - Five files on one message, thumbnails while uploading', () => {
+        const filename = 'huge-image.jpg';
+        cy.get('#centerChannelFooter').find('#fileUploadInput').
+            attachFile(filename).
+            attachFile(filename).
+            attachFile(filename).
+            attachFile(filename).
+            attachFile(filename);
+        cy.get('.file-preview__container').find('.post-image').each(($el, idx) => {
+            if (idx === 4) {
+                cy.wrap($el).should('not.be.visible');
+            } else {
+                cy.wrap($el).should('be.visible');
+            }
+        });
+        cy.get('.file-preview__container').scrollTo('right');
+        cy.get('.file-preview__container').find('.post-image').each(($el, idx) => {
+            if (idx < 3) {
+                cy.wrap($el).should('not.be.visible');
+            } else {
+                cy.wrap($el).should('be.visible');
+            }
+        });
+        cy.postMessage('{enter}');
+        cy.get('[data-testid=postView]').find('.post-image').should('have.length', 5);
```

Comment:
nit: Good as is but preferred,
```suggestion
        cy.findByTestId('postView').find('.post-image').should('have.length', 5);
```